src/js/Content/Features/Community/Badges/FBadgeCalculations.js

```diff
@@ -158,38 +151,35 @@ export default class FBadgeCalculations extends Feature {
 
     _countDropsFromDOM(dom) {
 
-        for (const node of dom.querySelectorAll(".badge_title_stats_drops .progress_info_bold")) {
```

Comment:
This can't be dropped and should probably be noted in a comment.

Take this for example:
![image](https://user-images.githubusercontent.com/14999077/113288195-e9f5bd80-92ee-11eb-928d-0091338fd4e4.png)

The marked text is the text content of a node with the `progress_info_bold` class, therefore a match on this would result in `count === 25`

Reply:
oops, didn't think of that, fixed and added a comment